common/dynamicconfig/constants.go

```diff
@@ -1919,6 +1919,9 @@ const (
 	// Value type: Bool
 	// Default value: true
 	EnableShardIDMetrics
+
```

Comment:
can you add the comments as the other constants have? it's kind of redundant with below definition but better to be consistent until we come up with a better structure to define these dynamicconfigs

Reply:
Actually, I'm planning to remove those comments.